src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -257,24 +265,30 @@ export class CalciteDropdown {
     return this.items.indexOf(e);
   }
 
-  private getFocusableElement(item) {
-    const target =
-      item && item.attributes.isLink
-        ? item.shadowRoot.querySelector("a")
-        : (item as HTMLCalciteDropdownItemElement);
-    target.focus();
+  private getFocusableElement(item): void {
+    if (!item) {
+      return;
+    }
+
+    const target = item.attributes.isLink
+      ? item.shadowRoot.querySelector("a")
+      : (item as HTMLCalciteDropdownItemElement);
+
+    focusElement(target);
   }
 
-  private openCalciteDropdown(e) {
+  private openCalciteDropdown() {
     this.active = !this.active;
-    // if invoked by key, focus item, and accomodate animation time
-    if (!e.detail && e.type !== "mouseenter") {
-      setTimeout(() => this.focusFirstItem(), 50);
-    }
+    const animationDelayInMs = 50;
```

Comment:
is there a reason you pulled this out into a variable?

Reply:
Yes, to get rid of the comment. This can help avoid stale comments when refactoring.

Reply:
botcha